### PR TITLE
CI: Enable Travis-CI for linux-ppc64le

### PR DESCRIPTION
**Pull request by nitkon**
FIxes: #365

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

**Comment by katabuilder** [comment]
PSS Measurement:
Qemu: 160022 KB
Proxy: 4589 KB
Shim: 8953 KB

Memory inside container:
Total Memory: 2045972 KB
Free Memory:  1998120 KB

**Comment by jodh-intel** [comment]
Oh nice @nitkon! I didn't realise this would be hosted on the main Travis site! ;)

Our [static-checks.sh](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh) is failing the CI for ppc64el though due to a few lint issues. Once they're fixed,

lgtm

**Comment by bergwolf** [comment]
lgtm! Please fix lint errors though so that this does not break CI.

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ sudo: required
 dist: trusty
```
Comment (grahamwhaley) [inline .travis.yml]: nit: feel free to add your company to the copyright list if you want ;-)

File: .travis.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ sudo: required
...
 
 language: go
+os:
+- linux
```
Comment (grahamwhaley) [inline .travis.yml]: I guess this is enabling x86 linux as well as ppc64 - I think that is probably the right thing, so we can sanity check that the Travis builds are working on both (to try and distinguish between architecture specific issues and generic issues).

File: .travis.yml (hunk 1 above)
Comment (nitkon) [reply .travis.yml]: @grahamwhaley : Added. Thanks!

File: .travis.yml (hunk 2 above)
Comment (nitkon) [reply .travis.yml]: @grahamwhaley : Updated my commit message accordingly.

**Comment by katabuilder** [comment]
PSS Measurement:
Qemu: 159798 KB
Proxy: 6619 KB
Shim: 11025 KB

Memory inside container:
Total Memory: 2045972 KB
Free Memory:  1997996 KB

**Comment by jodh-intel** [comment]
We have all the approvals now, so we just need the lint errors to be fixed.

**Comment by nitkon** [comment]
@jodh-intel : All green!!!

**Comment by jodh-intel** [comment]
In the immortal words of Captain Barnacles, "Let's do this!"...

**Pull request merged**